api/src/controllers/common.ts

```diff
@@ -0,0 +1,16 @@
+import { ILogin } from "../models/commonRequest";
+import services from "../services";
+import { Request, Response } from "express";
+import * as response from "../utils/response";
```

Comment:
- không import như vậy, bên utils/response phải dùng export default chớ ko phải export
```js
const a = () => {
  // ...
}

const b = () => {
  // ...
}

export default { a, b }
```

```js
import x from 'x'

x.a()
x.b()
```

- fix hết lại toàn bộ project

Reply:
done